modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -931,7 +931,22 @@
 			<description>
 				Random range, any floating point value between [code]from[/code] and [code]to[/code].
 				[codeblock]
-				prints(rand_range(0, 1), rand_range(0, 1)) # Prints e.g. 0.135591 0.405263
+				prints(randf_range(0, 1), randf_range(0, 1)) # Prints e.g. 0.135591 0.405263
```

Comment:
Nitpick, but `randf_range(0, 1)` is not a very useful example as it's the same behavior as `randf()`. Something like `randf_range(-10, 10)` could be more interesting maybe.

Reply:
Fixed